usaspending_api/common/helpers/fiscal_year_helpers.py

```diff
@@ -46,7 +48,7 @@ def generate_fiscal_year(date):
     return year
 
 
-def generate_fiscal_month(date):
+def generate_fiscal_period(date):
```

Comment:
I tend to prefer "month" over "period" since the "time period" could be in years, quarters, or months. I understand there are inconsistencies with source data, so any normalization we can do it appreciated